docs/storage/azure-storage.md

```diff
@@ -17,17 +18,25 @@ Cloud-native apps often require scalable storage solutions that provide capabili
 
 [!INCLUDE [aspire-prereqs](../includes/aspire-prereqs.md)]
 
-## Set up the Azure Storage Account
+## Set up the Azure Storage account
 
-For this quickstart, you'll need data contributor access to an Azure Storage Account with a blob container and storage queue. Ensure you have the following resources and configurations available:
+For this quickstart, you'll need data contributor access to an Azure Storage account with a blob container and storage queue. Ensure you have the following resources and configurations available:
```

Comment:
This part doesn't look right on the rendered preview on some of the zone pivots - I think this verbiage or zone pivot section needs to be adjusted.

Reply:
Fixed